aws/gpu_tree_shap/gpu_tree_shap.ipynb

```diff
@@ -0,0 +1,493 @@
+{
```

Comment:
<p>I'd just change the title to "GPU-accelerated Tree SHAP on AWS" since XGBoost can also get updated and this title was mainly used for the first release</p><p> </p><p>We don't need a link to the papers here, can link Rory's blog instead since that has all the details</p><p> </p><p>The NYC Taxi data looks more interesting, however the spreadsheet is missing SHAP time for this dataset. Once you add that we can decide which one to use here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cloud-ml-examples/pull/149/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
ok, I'll work on getting the SHAP values right now. my only concern is that for the sklearn data, we generate predictions/SHAP values for every sample in the dataset (~20k) whereas NYC taxi has 6.3M, so to avoid time-outs I may have to generate them for just a portion of the data. 

another option is just generating the prediction and SHAP values for a single example at a time, which is what AWS does in their example notebook: https://github.com/aws/amazon-sagemaker-examples/blob/master/introduction_to_amazon_algorithms/xgboost_abalone/xgboost_inferenece_script_mode.ipynb

Reply:
Let's try the NYC data first, we still want to use the maximum number of rows to be able to compare training as well